Further usecases (#13)
ElektraInitiative/PermaplanT: doc/usecases/translation.md

# Use Case: Translation

## Summary

- **Scope:** Translation
- **Level:** User Goal
- **Actors:** App User

> - **Brief:** The user can translate the app's interface and text into English or German

Reviewer: ```suggestion
- **Brief:** The user can choose the app's interface to be in English or German
```

- **Status:** Draft

## Scenarios

- **Precondition:**
  The user has opened the app and wants to change the language of the interface.
- **Main success scenario:**

>   The user successfully translates the app's interface and text into English or German by selecting the desired language from the app's menu.

Reviewer: Can you additionally guess the language from the browser infos? (But language should be changeable in the app's settings anyway.)

Author: Should be possible, will add it to the decision
